sensor-core/src/main/java/com/mindlamp/Accelerometer.java

```diff
@@ -6,9 +6,6 @@
 import android.content.Context;
```

Comment:
There are many `.DS_Store` files that should not be checked into the repository, please add it to the `.gitignore` and remove them.

Reply:
.gitignore file updated in sensor-core

Reply:
removed .DS_Store form git